app/js/lib/foxbox/foxbox.js

```diff
@@ -23,55 +19,33 @@ const p = Object.freeze({
   db: Symbol('db'),
   net: Symbol('net'),
   boxes: Symbol('boxes'),
-  isPollingEnabled: Symbol('isPollingEnabled'),
-  nextPollTimeout: Symbol('nextPollTimeout'),
   webPush: Symbol('webPush'),
-
-  // Private methods.
-  fetchServices: Symbol('fetchServices'),
-  getServiceInstance: Symbol('getServiceInstance'),
-  hasDoorLockChannel: Symbol('hasDoorLockChannel'),
+  api: Symbol('api'),
 });
 
-/**
- * Compare 2 objects. Returns true if all properties of object A have the same
- * value in object B. Extraneous properties in object B are ignored.
- * Properties order is not important.
- *
- * @param {Object} objectA
- * @param {Object} objectB
- * @return {boolean}
- */
-const isSimilar = (objectA, objectB) => {
-  for (let prop in objectA) {
-    if (!(prop in objectB) || objectA[prop] !== objectB[prop]) {
-      return false;
-    }
-  }
-
-  return true;
-};
-
 export default class Foxbox extends Service {
   constructor() {
     super();
 
     // Private properties.
     this[p.settings] = new Settings();
     this[p.db] = new Db();
-    this[p.net] = new Network(this[p.settings], (foxboxOnline) => {
-      this._dispatchEvent('box-online', foxboxOnline);
+
+    this[p.net] = new Network(this[p.settings]);
+    this[p.net].on('online-state-changed', (online) => {
```

Comment:
>> I know I suggested preterite verb, but maybe name only event names are better.

:) Yeah, I have the same feeling honestly, just followed what we agreed on previously ;) So, remove ```preterite verb`` part and we're good? I'm fine with this.

>> Likewise, in message-received, a message related event is always going to be about receiving a message, so message looks good enough to me.

Not always :) In message app we have ```message-received``` ```message-sent``` ```message-failed-to-sent``` :) But in this case just ```message``` should be ok.